src/sizzle.js

```diff
@@ -651,6 +672,30 @@ setDocument = Sizzle.setDocument = function( node ) {
 			!el.querySelectorAll( ":scope fieldset div" ).length;
 	} );
 
+	// Support: Chrome 105+, Firefox 104+, Safari 15.4+
```

Comment:
```suggestion
	// Support: Chrome 105+, Firefox 104+, Safari 15.4+, IE 6 - 11+, Edge 12 - 18+
```
Plus, I should mention IE & Edge as ones not supporting the API at all.